x/liquidfarming/spec/02_state.md

```diff
@@ -14,10 +14,10 @@ type LiquidFarms struct {
 
 // LiquidFarm defines liquid farm.
 type LiquidFarm struct {
-	PoolId           uint64        // the pool id
-	MinDepositAmount sdk.Int       // the minimum deposit amount; it allows zero value
-	MinBidAmount     sdk.Int       // the minimum bid amount; it allows zero value
-	AuctionPeriod    time.Duration // default value is 12 hours
+	PoolId        uint64  // the pool id
+	MinFarmAmount sdk.Int // the minimum farm amount; it allows zero value
+	MinBidAmount  sdk.Int // the minimum bid amount; it allows zero value
+	FeeRate       sdk.Dec // default value is 12 hours
```

Comment:
Can you update the comment for `FeeRate`?

`// the fee rate that deducts from auction winner's rewards; default value is 0`

Reply:
Thanks! Modified!